business/metrics.go

```diff
@@ -226,3 +226,31 @@ func createStatsMetricsLabelsBuilder(q *models.MetricsStatsQuery) *MetricsLabels
 	}
 	return lb
 }
+
+func (in *MetricsService) GetControlPlaneMetrics(q models.IstioMetricsQuery, scaler func(n string) float64) (models.MetricsMap, error) {
+	metrics := make(models.MetricsMap)
+
+	h := in.prom.FetchHistogramRange("pilot_proxy_convergence_time", "", "", &q.RangeQuery)
```

Comment:
The `pilot_proxy_convergence_time`  metric is not on the [required metric list](https://kiali.io/docs/faq/general/#requiredmetrics) and needs to be added.  As such it is removed in our [thinning config](https://kiali.io/docs/configuration/p8s-jaeger-grafana/prometheus/#metric-thinning).  We will need to fix the recommended thinning config and make the appropriate kiali.io changes.

Reply:
Thanks, this is addressed on: https://github.com/kiali/kiali.io/pull/582